ascii_art.py

```diff
@@ -0,0 +1,49 @@
+import PIL.Image
+
+# ascii characters used to build the output text
+ASCII_CHARS = ["@", "#", "S", "%", "?", "*", "+", ";", ":", ",", "."]
+
+# resize image according to a new width
+def resize_image(image, new_width=100):
+    width, height = image.size
+    ratio = height/width/1.65
+    new_height = int(new_width * ratio)
+    resized_image = image.resize((new_width, new_height))
+    return(resized_image)
+
+# convert each pixel to grayscale
+def grayify(image):
+    grayscale_image = image.convert("L")
+    return(grayscale_image)
+    
+# convert pixels to a string of ascii characters
+def pixels_to_ascii(image):
+    pixels = image.getdata()
+    characters = "".join([ASCII_CHARS[pixel//25] for pixel in pixels])
+    return(characters)    
+
+def main(new_width=100):
+    # attempt to open image from user-input
+    path = input("Enter a valid pathname to an image:\n")
```

Comment:
Can you please use argparse for this?
Also, try integrating tqdm for progress bars

Reply:
Sure sir, I shall modify the code and update you with the progress!😊